jupyterhub_singleuser_profiles/images.py

```diff
@@ -37,6 +35,22 @@ def append_option(self, image, result, last_image):
                 selected = "selected=selected"
             result.append("<option value='%s' %s>%s</option>" % (image_tag, selected, image_tag))
 
+    def get_list(self):
```

Comment:
Can you refactor this and append_option so that the same code is there only once?